### PR TITLE
{chem}[foss/2017a] NAMD v2.12

### PR DESCRIPTION
**Pull request by henkela**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/n/NAMD/NAMD-2.12-foss-2017a-mpi.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name = 'NAMD'
+version = '2.12'
+versionsuffix = '-mpi'
+
+homepage = 'http://www.ks.uiuc.edu/Research/namd/'
+description = """NAMD is a parallel molecular dynamics code designed for high-performance simulation of
+ large biomolecular systems."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'opt': True, 'pic': True}
```
Comment (boegel) [inline easybuild/easyconfigs/n/NAMD/NAMD-2.12-foss-2017a-mpi.eb]: no `'usempi': True`, but still an MPI-abled build? cfr. `versionsuffix`

Comment (henkela) [inline easybuild/easyconfigs/n/NAMD/NAMD-2.12-foss-2017a-mpi.eb]: Sorry. I'll check

**Comment by akesandgren** [comment]
Should sync this with the updated namd easyblock and my corresponding namd easyconfig.

**Comment by henkela** [comment]
Saw it yeterday. Sorry for the mess. Will clean it up asap.

**Comment by henkela** [comment]
@akesandgren 
I added FFTW as a dependency like in NAMD-2.10b1-ictce-5.5.0-mpi.eb but left out the 'opt': true beacuse it's default now, isn't it?

**Comment by boegel** [comment]
@henkela `opt` is indeed enabled by default, this looks good, I'll get it tested

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2114.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4df7fd71eaae2d8e54d38f4db003912a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2532.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f0a78827b9debc880f420007f3dc163e for a full test report.

**Comment by boegel** [comment]
Tested on top of https://github.com/hpcugent/easybuild-easyblocks/pull/1173, good to go.

**Pull request merged**